NEWS.md

```diff
@@ -1,3 +1,7 @@
+# hint 1.89.3
```

Comment:
Since this is a change in functionality rather than a bug fix, we'd generally make this a minor version rather than a patch, so would expect this to be 1.90.0